src/plugins/usage_collection/server/config.ts

```diff
@@ -11,6 +11,11 @@ import { PluginConfigDescriptor } from 'src/core/server';
 import { DEFAULT_MAXIMUM_WAIT_TIME_FOR_ALL_COLLECTORS_IN_S } from '../common/constants';
 
 export const configSchema = schema.object({
+  usageCounters: schema.object({
+    enabled: schema.boolean({ defaultValue: true }),
+    retryCount: schema.number({ defaultValue: 1 }),
+    bufferDebounceMs: schema.number({ defaultValue: 30 * 1000 }),
```

Comment:
Maybe 30 seconds is too much to wait? I think this is great to merge counters called very soon in a row. But 30s seems like a risk too high to losing data, don't you think?

Reply:
I agree. Updated to `5s` since it becomes more unlikely that the same counter gets spammed after the first few seconds.